MovieSharing/ImageObject.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ImageObject.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 01.10.21.
+//
+
+import Foundation
+
+struct ImageObject: Decodable {
+    
+    // MARK: - Types
+    
+    enum CodingKeys: String, CodingKey {
+        case imageUrl = "url"
+        case imageWidth = "width"
+        case imageHeight = "height"
+    }
```

Comment:
- [ ] I would remove these CodingKeys as you already have `Image` in the struct name, so just `url`, `width` and `height` are fine